### DISCUSSION TITLE
[Coreshop Datahub] Price import from CSV with Datahub

### DISCUSSION THREAD

**Discussion by twin-elements** [pimcore/pimcore#16907] [Q&A / Support] [answered]
Hi.
Does Coreshop somehow allow you to import product prices using Datahub import?

**Comment by dpfaffenbauer** [comment, marked as answer]
Hey, you opened the same question in the CoreShop Forum and here. Usually one is enough, so follow up here please: https://github.com/coreshop/CoreShop/discussions/2596
